lib/config.coffee

```diff
@@ -67,4 +67,10 @@ module.exports = {
     default: false
     description: "Mute 'Latest backup is already applied' message"
     order: 14
+  removePackage:
```

Comment:
Can you please add the trailing `s` to make it plural. Since the title is "the same" afterwards you can remove it (it is automatically generated by separating the capitalized words of the key).